This part describes the workflow to generate the heatmaps in Figure 2. Based on the kaiju metagenomics pipeline output, the process is as following:
![scheme](https://github.com/LandthalerLab/wastewater_virome/assets/57175100/f8b5eb6c-4133-493d-88b8-a653133cfdc9)

1. The kaiju pipelines identifies taxonomy IDs found in the sequencing data. The first step is to download all the fasta files belonging to the taxonomy IDs in the kaiju output file from a specific datasets. These output files for the Berlin wastewater RNA and DNA data are on the GEO entry (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228220). The kaiju output files of the three other datasets, California (…), India (Stockdale et al. 2023), and Baseline (Nieuwenhuijse et al. 2020) will be available there soon. This is done using the bash script mapvirus_part1.sh, which requires the location of the kaiju output file and the path to the NCBI entrez direct tools. The search terms used in the paper were: Astroviruses – "astrovir", enteroviruses – "enterovir", noroviruses – "norovir", adenoviruses – "adenovir". Adjust the parameters and run the script with "bash mapvirus_part1.sh". Depending on the connection speed and the number of entries, this takes a few minutes to less than an hour.
2. The second step is to select the sequences that should be used for the meeting. For every taxonomy ID, the sequences between $lowerbound and $upperbound are selected.  
